### PR TITLE
currying the output of wrap

### LINKED ISSUE #725: `R.wrap` should make curried function
Kind of strange, isn't it stated that every function in Ramda is autmatically curried?

```js
var wrappedAdd = R.wrap(R.add, function(add, x, y) {
  console.log('adding %s and %s', x, y); 
  return add(x, y); 
});

wrappedAdd(1) // "adding 1 and undefined" => NaN
wrappedAdd(1, 2) // "adding 1 and 2" => 3
```

I particularly interested in wrapping functions for debugging as shown in this example.

### PR DESCRIPTION
**Pull request by CrossEye**
Fixes #725.

(Let's see if it works this time!)

### CODE REVIEW

**Comment by davidchambers** [comment]
:deciduous_tree:

File: src/wrap.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ var arity = require('./arity');
  *
```
Comment (anton-rudeshko) [inline src/wrap.js]: Could you please add an example of currying?

Comment (CrossEye) [inline src/wrap.js]: @anton-rudeshko: I guess I'm willing to, if you feel strongly about it.  I don't see much of a need.  Part of it is that this function is not one I use often, so I'm not particularly invested in it.  Another part is that I'm (still!) generally less interested in inline documentation.  :-)

Do you have a suggested short example?

Comment (anton-rudeshko) [inline src/wrap.js]: Hmm, no example right now. Will do a PR if any come to mind. Thanks for the fix.

**Pull request merged**